### PR TITLE
Prepare gem release as "carrierwave-activejob"

### PR DESCRIPTION
**Pull request by yoshih**
References https://github.com/denkungsart/castupload/issues/5019
and https://github.com/denkungsart/castupload/pull/5044#discussion_r488818191

Changes:
* adapt gemspec

### CODE REVIEW

**Comment by yoshih** [comment]
@andreas-venturini 
Is it necessary to rename the .gemspec file also? Namespaces Carrierwave::Backgrounder haven't changed.

File: carrierwave_backgrounder.gemspec [hunk 1]
```diff
@@ -3,12 +3,12 @@ $:.push File.expand_path("../lib", __FILE__)
 require "backgrounder/version"
```
Comment (fschwahn) [inline carrierwave_backgrounder.gemspec:3]: Rename the gemspec file, and add a file `lib/carrierwave/activejob.rb` which just does `require "carrierwave_backgrounder"`. I think that's necessary that the gem can be required without manually specifying `require` in the Gemfile.

Comment (fschwahn) [reply carrierwave_backgrounder.gemspec:3]: You can test if this actually works by pointing your Gemfile to this branch.

Comment (yoshih) [reply carrierwave_backgrounder.gemspec:3]: I mounted my local branch and uploaded a showreel video, completed without error message. But when I enter `Carrierwave::Backgrounder` in the console, it says "uninitialized constant".

Comment (yoshih) [reply carrierwave_backgrounder.gemspec:3]: When I remove the require statement in the gem, I get an error in showreel manager (undefined method `processed?`), so I think it works.

Comment (fschwahn) [reply carrierwave_backgrounder.gemspec:3]: Try `CarrierWave::Backgrounder`? But if there's no error when uploading videos it has to work.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## unreleased
```
Comment (fschwahn) [inline CHANGELOG.md]: Actually as we never released 0.5.0, this can be released as 0.5.0, no need for a new version,

**Review by fschwahn — APPROVED** [review]
Seems fine, you can release the gem yourself.

File: carrierwave-activejob.gemspec [hunk 3]
```diff
@@ -3,12 +3,12 @@ $:.push File.expand_path("../lib", __FILE__)
...
   s.version     = CarrierWave::Backgrounder::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Larry Sprock"]
-  s.email       = ["larry@lucidbleu.com"]
-  s.homepage    = "https://github.com/lardawge/carrierwave_backgrounder"
+  s.authors     = ["Fabian Schwahn"]
+  s.email       = ["fabian.schwahn@gmail.com"]
+  s.homepage    = ""
   s.licenses    = ["MIT"]
   s.summary     = %q{Offload CarrierWave's image processing and storage to a background process using ActiveJob}
```
Comment (andreas-venturini) [inline carrierwave-activejob.gemspec:14]: we need to update the allowed push host configuration (copy this from another gem of ours)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ## 0.5.0
+  * Adapt gemspec to new name carrierwave-activejob and prepare to release
```
Comment (andreas-venturini) [inline CHANGELOG.md]: *Rename gem to carrierwave-activejob*

File: lib/carrierwave/activejob.rb [hunk 5]
```diff
@@ -0,0 +1 @@
+require "carrierwave_backgrounder"
```
Comment (andreas-venturini) [inline lib/carrierwave/activejob.rb:2]: Might be obvious, but let's add a comment here indicating that we do this to avoid having to rename internals.

**Pull request merged**